app/views/admin/users/index/_controls.html.erb

```diff
@@ -1,15 +1,59 @@
+<%= javascript_packs_with_chunks_tag "admin/members/memberControls", defer: true %>
+
 <% if FeatureFlag.enabled?(:member_index_view) %>
   <% if params[:controller] == "admin/users" %>
-    <%= form_with url: admin_users_path, method: :get, local: true, class: "flex flex-col m:flex-row gap-3 m:items-center p-3" do |f| %>
-      <div class="crayons-field flex-1 flex-row items-center gap-2">
-        <%= f.text_field :search, value: params[:search], class: "crayons-textfield mt-0", placeholder: "Search member...", aria: { label: "Search member by name, username, email, or Twitter/GitHub usernames" } %>
+
+    <!-- Small screen layout -->
+    <div class="flex justify-between my-3 l:hidden">
+      <div>
+        <button id="expand-search-btn" class="c-btn c-btn--icon-alone" aria-label="Expand search" aria-expanded="false" aria-controls="search-users">
+          <%= crayons_icon_tag("search", aria_hidden: true) %>
+        </button>
+        <button id="expand-filter-btn" class="c-btn c-btn--icon-alone" aria-label="Expand filter" aria-expanded="false" aria-controls="filter-users">
+          <%= crayons_icon_tag("filter", aria_hidden: true) %>
+        </button>
       </div>
-      <div class="crayons-field flex-row items-center gap-2">
-        <%= f.select :role, options_for_select(Role::ROLES, params[:role]), { include_blank: true, prompt: "Filter by role..." }, class: "crayons-select mt-0" %>
+      <%= paginate @users, theme: "admin", scope: @users, label: "Paginate users" %>
```

Comment:
When we start using these controls on other pages (e.g. Invitations) we won't be able to simply call `paginate @users`, as it might be e.g. `@invitations`. I think the pagination _needs_ to live in this file though, for layout reasons (and I guess also thematically it _is_ part of the 'controls'). 

In short, when we tackle adding the search functionality to those other views, we will need to refactor a little to allow the pagination to target the right records